http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -108,6 +108,7 @@ export NOTEBOOKS_DIR=$(notebooksDir)
 export WELDER_SERVER_NAME=$(welderServerName)
 export WELDER_DOCKER_IMAGE=$(welderDockerImage)
 export WELDER_ENABLED=$(welderEnabled)
+export IS_GCE_FORMATTED=$(isGceFormatted)
```

Comment:
no longer need `PD_ENABLED` because of @Qi77Qi's changes (now every gce runtime will have a user disk)